woocommerce/woocommerce-ios
# Edit Product: implemented the new product settings screen

Fixes #1821 

## Description 
The new Edit Product will allow users to add some extra settings. This PR implements the new Product Settings screen, which will be shown after pressing the new voice "Product Settings" inside the "more" action sheet #1819.

## Changes
- Implemented `ProductSettingsTableViewModel`
- Implemented the new `ProductSettingsViewController` + .xib
- Present the `ProductSettingsViewController` from the `ProductFormViewController` action sheet.

## Screenshot
<img width="300" alt="screenshot" src="https://user-images.githubusercontent.com/495617/74269380-abc3c600-4d09-11ea-9dd8-bb82d95e9490.png">

## Testing
1) Navigate to a product detail
2) Tap the "more" button in the navbar
3) Tap the Product Settings button inside the action sheet
4) Make sure you are able to see the new Product Settings screen. All the cells are without connected actions at the moment (part of another PR).

Update release notes:
